dotnet/src/SemanticKernel.Abstractions/AI/FunctionChoiceBehaviors/NoneFunctionChoiceBehavior.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+
+namespace Microsoft.SemanticKernel;
+
+/// <summary>
+/// Represents <see cref="FunctionChoiceBehavior"/> that provides either all of the <see cref="Kernel"/>'s plugins' functions to the AI model to call or specific ones.
```

Comment:
Consider something like this:

`Represents a <see cref="FunctionChoiceBehavior"/> that advertises some or all <see cref="Kernel"/>'s plugins' functions to the model but instructs it not to call any of them. The model may use the advertised function in the response it generates. E.g. the model may describe which functions it would call and with what parameter values. This response is useful if the user should first validate what functions the model will use.`

Can you include a Concept sample which shows this behaviour in action and include a sample response from the LLM so developers will see the benefit of using this approach.

Reply:
The class XML comment is updated to follow your example. The concept sample will be linked to the class and the other ones when later when we have them.